model/client4.go

```diff
@@ -3701,8 +3701,8 @@ func (c *Client4) GetWebappPlugins() ([]*Manifest, *Response) {
 
 // ActivatePlugin will activate an plugin installed.
```

Comment:
Should we drop these comments now?

Reply:
When we actually release plugins out of beta, yes. I don't think we are doing that on the next release.